src/history/history.rs

```diff
@@ -396,7 +396,7 @@ impl History {
             }));
         }
 
-        if fuzzy > 0 {
+        if fuzzy > 0 && result_sort != &ResultSort::LastRun {
```

Comment:
In my opinion, since there are only two options for result sort, it does not matter. Using == ResultSort::Rank or != ResultSort::LastRun returns the same result. It will only matter when there are more result sort options and if they will use fuzzy search or not. @cantino what are your thoughts here?

Reply:
I think this is alright.